### PR TITLE
[Feat] Added HebeSwap Oracle for Ethereum Classic

### PR DESCRIPTION
**Pull request by a-singh09**
Added HebeSwap Oracle for getting price feed for `ETC/USDT`.
Deploy script has also been added.

### CODE REVIEW

File: src/HebeSwapOracle.sol
Comment (yogesh0509) [inline src/HebeSwapOracle.sol:1]: We need price feed from USDT/ETH for our frontend. Can you look at any of the inverting oracle and try replicating the same with Hebeswap.

File: env/example.env [hunk 1]
```diff
@@ -18,4 +18,12 @@ DJED_DECIMALS=18
...
 
 DATA_FEED_ADDRESS=0xAB594600376Ec9fD91F8e885dADF0CE036862dE0 # MATIC/USD: Polygon Mainnet
+DJED_DECIMALS=18
+
+###############
+# HebeSwap Oracle #
+###############
+
+DATA_FEED_ADDRESS=0xBe004D5A567f7Fb0c59D451B583E925fC96d7804 # ETC/USDT: Ethereum Classic Testnet (MORDOR)
+HEBESWAP_DECIMALS=
```
Comment (ceilican) [inline env/example.env:28]: Why is this empty?

File: remappings.txt [hunk 2]
```diff
@@ -3,4 +3,5 @@ solmate/=lib/solmate/src/
 forge-std/=lib/forge-std/src/
 @openzeppelin/=lib/openzeppelin-contracts/
 @api3dao=lib/contracts/
-@chainlink/contracts/=node_modules/@chainlink/contracts
\ No newline at end of file
+@chainlink/contracts/=node_modules/@chainlink/contracts
+@Oracle=lib/hebeswap-contract/
```
Comment (ceilican) [inline remappings.txt:7]: Let's call this "hebeswap" instead of "Oracle".

File: scripts/deployHebeSwapOracle.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (ceilican) [inline scripts/deployHebeSwapOracle.sh:1]: Don't we need a `deployHebeSwapInvertingOracle.sh` script as well?

File: env/example.env (hunk 1 above)
Comment (a-singh09) [reply env/example.env:28]: I have kept it empty just like API3_DECIMALS in API3 Oracle. HEBESWAP_DECIMALS value is 18, should 
I add it?

File: remappings.txt (hunk 2 above)
Comment (a-singh09) [reply remappings.txt]: Done

File: scripts/deployHebeSwapOracle.sh (hunk 3 above)
Comment (a-singh09) [reply scripts/deployHebeSwapOracle.sh:1]: Done

**Pull request merged**